src/External/NBitcoin/Network.cs

```diff
@@ -129,6 +129,7 @@ public abstract class Network
         /// <summary>
         /// The default port on which SignalR broadcasts for this network.
         /// </summary>
+        [Obsolete("SignalR has been migrated into WebHost and use same port as API.")]
```

Comment:
Can't we just delete this? whats the dependency?

Reply:
All nodes set this value. Just wanted to avoid getting compile error, but we could delete it.

Reply:
ah I see, in my opinion its ok, thats the code of upgrade

Reply:
We'll remove it quickly in a future refactoring =)